### PR TITLE
Sample Design of AdityBeautyCare

### PR DESCRIPTION
**Pull request by ShreenidhiHD**
I have added One  business project and another UI project With clean architecture design with end to end implementation from UI to Service to Repositories

### CODE REVIEW

**Review by smartdusttechnologies — COMMENTED** [review]
please do the folowing:
1. remove commented codes.
2. put proper comment everywhere
3. put reason for public and private methods.

File: AditiBeautyCare/AditiBeautyCare.Business/Common/Sample/RequestResult.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AditiBeautyCare.Business.Common.Sample;
+namespace AditiBeautyCare.Business.Common.Sample
+{
+    public class RequestResult<T> : ResultBase
```
Comment (smartdusttechnologies) [inline AditiBeautyCare/AditiBeautyCare.Business/Common/Sample/RequestResult.cs:7]: please put comments here to explain the purpose of the method

Comment (ShreenidhiHD) [reply AditiBeautyCare/AditiBeautyCare.Business/Common/Sample/RequestResult.cs:4]: added the comment as asked

**Review by smartdusttechnologies — APPROVED** [review]
Looks good for now. Raise another pull request for remaining comments.

**Pull request merged**